src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -198,4 +198,17 @@
   <data name="ERROR_CreatingVcxRequest" xml:space="preserve">
     <value>[CFamily Analysis] Error creating VCX request. File: {0}, Error: {1}</value>
   </data>
+  <data name="MsgHandler_ReportInvalidInput" xml:space="preserve">
+    <value>[CFamily Analysis] Internal error calling the analysis subprocess: {0}</value>
```

Comment:
The "internal error" messages don't log the full file path. I decided not to log it because  is logged in the "analysis failed" message that comes immediately afterwards. The file path can be quite long, so omitting it makes it more likely the user can read the specific failure message without scrolling.